src/sources/kubernetes_logs/mod.rs

```diff
@@ -138,17 +138,17 @@ pub struct Config {
     /// a significant overhead.
     glob_minimum_cooldown_ms: usize,
 
-    /// A field to use to set the timestamp when Vector ingested the event.
+    /// This field sets the timestamp when an event is ingested.
     /// This is useful to compute the latency between important event processing
-    /// stages, i.e. the time delta between log line was written and when it was
+    /// stages. For example, the time delta between when a log line was written and when it was
     /// processed by the `kubernetes_logs` source.
```

Comment:
```suggestion
    /// Overrides the name of the log field used to add the ingestion timestamp to each event.
    ///
    /// This is useful to compute the latency between important event processing
    /// stages. For example, the time delta between when a log line was written and when it was
    /// processed by the `kubernetes_logs` source.
    ///
    /// By default, the [global `log_schema.timestamp_key` option][global_timestamp_key] is used.
    ///
    /// [global_timestamp_key]: https://vector.dev/docs/reference/configuration/global-options/#log_schema.timestamp_key
```

Reply:
I couldn't commit this in GH UI because it gave me an error about not being able to make changes on deleted lines, so I made the change locally and pushed it up.